### DISCUSSION TITLE
Add Bangladeshi Mobile operators

### DISCUSSION THREAD

**Discussion by mehediceps** [kyujin-cho/pixel-volte-patch#244] [Carrier-specific Experiences]
Hello,
Thank you for the project. I am from Bangladesh and we pixel phone user are facing network issue. Kindly add Bangladeshi mobile operators to your apps. Kindly let me know if I could do anything from my side.

Thank you and best regards.

**Comment by fillwithjoy1** [comment]
Elaborate what's the issue

**Comment by Synapse07** [comment]
Issue is, VOLTE does not work on carrier Grameenphone. Grameenphone VOLTE works on iphone and other non pixel devices.
On the flip side VOLTE works on carrier Airtel with pixel IMS.
